armi/reactor/converters/blockConverters.py

```diff
@@ -311,7 +311,8 @@ def convert(self):
                 soluteName, self.solventName, minID=self.specifiedMinID
             )
         solvent = self._sourceBlock.getComponentByName(self.solventName)
-        BlockConverter._verifyExpansion(self, self.soluteNames, solvent)
+        if solvent.__class__ is not components.DerivedShape:
```

Comment:
Can we add a test case for the scenario you ran into and make sure that the else condition is checked or tested? For example, if the solvent is not a derived shape should the block converter even have defined behavior? I'm not sure.

Reply:
Test added. `MultipleComponentMerger` had no tests. I test dissolving two components into a derived shape solvent (the case that was broken) and dissolving two components into a circle solvent (the only non-derived shape solvent with defined behavior). The tests are similar to existing tests `test_dissolveWireIntoCoolant` and `test_dissolveLinerIntoClad` for `ComponentMerger`.